docs/docs/reference/index.md

```diff
@@ -1,11 +1,56 @@
 # Reference
 
-This section of the Keptn documentation contains reference documentation for the various APIs and CRDs that are used in Keptn.
+Welcome to the Keptn documentation reference section, your comprehensive guide to the various Custom Resource Definitions (CRDs) and APIs used in Keptn.
```

Comment:
```suggestion
Welcome to the Keptn documentation reference section,
which provides comprehensive information about the Keptn Custom Resource Definitions (CRDs) and APIs used in Keptn.
```
I'm not crazy about calling reference material a guide since one of the foundational principles of tech writing is that guides and reference material are two different types of documentation.  And yeah, your terminology doesn't exactly contradict that but someone will probably wince so good to just avoid the problem.

Reply:
Alright I will commit your suggestion.